src/modules/AuthenticationModule/components/Login/Login.tsx

```diff
@@ -1,16 +1,84 @@
-import logo from "../../../../assets/PMS 3.svg";
+import React, { useContext } from 'react'
+import logo from '../../../../assets/PMS 3.png'
+import { Link, useNavigate } from 'react-router-dom'
+import { useForm } from 'react-hook-form';
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { AuthContext } from '../../../SharedModule/components/AuthContext/AuthContext';
 
+export default function Login() {
 
+  const {baseUrl} = useContext(AuthContext)
 
+  const { register, handleSubmit, formState: { errors } } = useForm()
 
-const Login = () => {
+  const navigate = useNavigate()
 
+  const onSubmit = async (data:any) => {
+    try {
+      const response = await axios.post(`${baseUrl}/Users/Login`, data)
+      localStorage.setItem('token' , response.data.token)
+      console.log(response);
+         toast.success("logged in successfully", {
+         autoClose: 3000,
+         hideProgressBar: true,
+         pauseOnHover: false
+         });
+      navigate('/dashboard')
+    } catch (error) {
+         toast.error(error.response.data.message, {
+         autoClose: 3000,
+         hideProgressBar: true,
+         pauseOnHover: false
+       });
+    }
+  }
 
   return (
     <>
-    login
+      <div className="login-bg  min-vh-100 d-flex justify-content-center align-items-center  ">
+        <div className="container ">
+          <div className="row justify-content-center align-items-center  ">
+            <div className="col-md-6 ">
+              <div className="logo text-center ">
+                <img src={logo} className='w-50' alt="" />
+              </div>
+              <div className="login-form p-5 my-2 rounded-4 ">
+                <div className="login-title mb-3">
+                  <p className='my-0 text-white small '>welcome to PMS</p>
+                  <h2 className='main-color' >
+                    Login
+                  </h2>
+                </div>
+                <form className='mt-4' onSubmit={handleSubmit(onSubmit)}>
+                  <div className="input-group my-3  ">
+                    <label className='main-color'>E-mail</label>
+                    <input type="text" className='w-100 p-2 ' placeholder='Enter Your E-mail'
+                      {...register('email', {required:'Email Is Required' ,   pattern: {
```

Comment:
Email, password and other validation will be used in multiple places. I consider making a file containing all needed validations and then use this validation in our application. This will make our code more concise.

You can look at this example:
https://github.com/AnasKhaled7/PMS-React/blob/main/src/lib/InputValidator.ts